eventuate-local-java-cdc-connector-test-util/src/main/java/io/eventuate/local/test/util/TestHelper.java

```diff
@@ -212,6 +221,24 @@ public void waitForEventInKafka(KafkaConsumer<String, byte[]> consumer, String e
     throw new RuntimeException("entity not found: " + entityId);
   }
 
+  //TODO: Use in all tests where is possible.
+  public MessageAssertion prepareBinlogEntryHandlerMessageQueue(BinlogEntryReader binlogEntryReader) {
+    ConcurrentLinkedQueue<MessageWithDestination> messages = new ConcurrentLinkedQueue<>();
+
+    binlogEntryReader.addBinlogEntryHandler(eventuateSchema,
+            "message",
+            new BinlogEntryToMessageConverter(idGenerator), new CdcDataPublisher<MessageWithDestination>(null, null, null, null) {
```

Comment:
This looks very complicated.
That's because addBinlogEntryHandler() expects a CdcDataPublisher, which is a class.
It would be better if it took an interface: https://github.com/eventuate-foundation/eventuate-cdc/issues/82
That way you could probably pass `messageAssertion::processMessage`